lib/mayaUsd/resources/ae/usd-shared-components/src/python/usdSharedComponents/common/resizable.py

```diff
@@ -127,6 +128,8 @@ def __init__(
         self._widget: QWidget = None
 
         self.loadPersistentStorage()
+        if self._contentSize < 0 and defaultSize > 0:
+            self._contentSize = defaultSize
```

Comment:
This avoids the lists to be very large by default. With the 80 passed in by the include/exclude panel, we get about 3 lines by default. Otherwise the lists a re way too big.